e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -119,7 +119,7 @@
               "InvocationSource": {
                 "value": "Cloud"
               },
-              "ServiceClientConnectionString": {
+              "IOT_HUB_CONNECTION_STRING": {
```

Comment:
Why this change?

Reply:
`ServiceClientConnectionString` is a variable used in Long Haul test. According to Philip's comment (https://github.com/Azure/iotedge/pull/2286#discussion_r365924786), he would like to use the variable `IOT_HUB_CONNECTION_STRING` in connectivity test instead.